### PR TITLE
Initialize frontend env

### PR DESCRIPTION
**Pull request by lukitoki1**
Initialize frontend env using create-react-app.

### CODE REVIEW

File: frontend/compute-system/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```
Comment (maciekkoz98) [inline frontend/compute-system/.gitignore:2]: I would add `.vscode` directory

File: frontend/compute-system/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (maciekkoz98) [inline frontend/compute-system/tsconfig.json]: maybe target es6

**Review by maciekkoz98 — COMMENTED** [review]
how about that?
I guess it's better to create a develop branch

**Comment by lukitoki1** [comment]
@maciekkoz98 I would stick to pulling this request to `master` branch for now. Creating `develop` branch and setting it as a defaut branch is the responsibility of the admin of this repo.

**Comment by maciekkoz98** [comment]
@lukitoki1 MAIN, remember #blm

**Comment by maciekkoz98** [comment]
Please remember to resolve review comments after a suggestion is completed. A good practise is also to write sth like `done` under the comment

**Comment by lukitoki1** [comment]
Already marked them both as resolved -.-

**Pull request merged**